### ISSUE TITLE
ARMA 3 Server - cannot connect to it

### ISSUE THREAD

**Issue by whoathere** [GameServerManagers/LinuxGSM#1290]
So I followed all the instructions at: https://gameservermanagers.com/lgsm/arma3server/

Modified the arma3server script with pertinent server details, modified the server.cfg file with server specific details (server name, password...etc).

I launch the game with ./arma3server start - it starts - I do ./armaserver details and I get all good details with at the bottom it says "ONLINE". Just above "ONLINE" it lists all the ports that the server is listening on. A netstat shows the ports indeed open.

I try to connect to it and nothing happens... ARMA3 acts like nothing is at the IP of the server when I use the IP and password.

Not sure what to do to troubleshoot this. Please help.

**Comment by UltimateByte** [comment]
https://github.com/GameServerManagers/LinuxGSM/wiki/Troubleshooting#connectivity-issues
https://github.com/GameServerManagers/LinuxGSM/wiki/Support

**Comment by whoathere** [comment]
I was able to connect to it but it had nothing to do with the links you provided. The issue was in the <server-name>.server.cfg file found in the /home/arma3server/arma3server/serverfiles/cfg folder - specifically the IP setting. I changed it to the internal IP of the linux server Im using and the server was then responsive.

So my question is if this setting is even necessary? Can I comment it out and it will still work?
